src/components/Button/Button.vue

```diff
@@ -509,13 +509,10 @@ export default {
 	}
 
 	&__icon {
-		height: $clickable-area;
```

Comment:
Why do you need to adjust the icon?

Reply:
As the icon had already the 44px height, the button couldn't shrink to `40px height + 2*2px border`, but stayed at `44px height + 2*2px border`. Thus i removed the fixed icon-area and moved the icon-spacing to the margin.